appuio-specifics-public.rst

```diff
@@ -50,6 +50,13 @@ Quotas and Limits
 The quotas are defined in the project size you ordered. The exact numbers can be found
 on the product site `APPUiO Public Platform <https://appuio.ch/public.html>`__
 
+Pod count Recommendations
+-------------------------
+The technical Red Hat limit is 250 pods per node. Kubernetes however recommends a maximum of 100 pods per node. 
+The maximum pod count per node is dependent on the workload. We recommend a maximum of 100 pods per node as you may face stability issues with a higher number of pods.
+
+For further information on the Kubernetes and Red Hat pod limits please check the official `Kubernetes <https://kubernetes.io/docs/setup/best-practices/cluster-large/.html>`__ and `Red Hat <https://docs.openshift.com/container-platform/3.7/scaling_performance/cluster_limits.html>`__ documentation.
```

Comment:
Something went wrong here with the link `.html` and OpenShift 3.7 is outdated! Please use 3.11!